### PR TITLE
Added files setupvpn.sh and cloud-bastion.ovpn

### PR DESCRIPTION
**Pull request by SergeyKa-cmd**
Added for further tests on Travis CI.

# Выполнено ДЗ № 5

 - [X] Main issue
 - [x] Task with *

## Roadmap:
 - Part 1
 + GCP account created. "bastionych" and "ellen-ripley" Vm instances are working correctly
 - Part 2
 + Prepared for two alternative shorthand path to connect with isolated "ellen-ripley" instance 
 - Part 3
 + Succesfull tests with connectivity to isolated "elen-ripley" instance using Pritunl VPN
 - Part 4
 + Prepared solution with trusted Certificate authority for Pritunl web interface

## How to launch project properly:
 - Prepare for ssh keys for Appuser user generating keys for GCP by using command:
 $ ssh-keygen -t rsa -f ~/.ssh/appuser -C appuser -P ""
 - Create instance with public IP with pre-installed VPN Pritunl web server (using attached setupvpn.sh file) and apply proper firewall inbound rules for this instance (Bastionych in that case) 
 - Create another instance without public IP connections which means to be isolated ("ellen-ripley" in that case) 
 - With generated file on Pritunl Web server (using  attached cloud-bastion.ovpn file) and OpenVPN client (used Pritunl Client in my case) connect to the isolated instance excluding to use instance with public IP addres (like host -> bastionych -> ellen-ripley) only using host -> ellen-ripley scheme.

## How to test:
 - Use attached cloud-bastion.ovpn file and Openvpn client on your host.
 - Try to connect with pre-installed ssh-key before and using command: 
 $ ssh -i ~/.ssh/appuser appuser@<host IP or hostname>
 - Ensure that your connection is established and secured on Pritunl Dashboard 35.187.97.192.nip.io

## PR checklist
 - [X] Выставил "cloud-bastion" с номером домашнего задания
 - [X] Выставил "cloud-bastion" с темой домашнего задания

### CODE REVIEW

**Comment by SergeyKa-cmd** [comment]
Dear Colleagues,
Please review current merge

File: Reamde.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        $ ssh appuser@ellen-ripley
+        
+    2. Way with using hops between hosts using -J option within one alias in ~/.bashrc file:
+      alias ellen-ripley='ssh -i ~/.ssh/appuser -A -J appuser@34.77.5.134 appuser@10.132.0.3'
+     # Run command:
+        
+        $ ellen-ripley
+-----------------------------------------------------------------------------------------------------------------------------------
+## How to test environment:
+### bastionych_IP = 34.77.5.134
```
Comment (pksmall) [inline Reamde.md]: wrong name - bastion_IP

File: Reamde.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+### ellen-ripley_IP = 10.132.0.3
```
Comment (pksmall) [inline Reamde.md]: again wrong name - someinterhost

**Review by pksmall — COMMENTED** [review]
PR is incorrect due to errors in vars for IP. Fix it. [Yes] Выставил "cloud-bastion"  - change Yes to X, and will get nice view.

File: Reamde.md (hunk 1 above)
Comment (SergeyKa-cmd) [reply Reamde.md]: Thank you pksmall for your review but it is real name of instances:
Screenshot from 2019-09-21 18-26-23
But I'll the change name on template

File: Reamde.md (hunk 2 above)
Comment (SergeyKa-cmd) [reply Reamde.md]: Got it!

**Comment by pksmall** [comment]
setupsvpn.sh -> setupvpn.sh

**Review by mrgreyves — APPROVED** [review]
Здравствуйте!) Проверил ваше дз. К сожалению бастион не доступен через https. В остальном все хорошо, дз принимаю!=)

**Comment by SergeyKa-cmd** [comment]
Dear mrgreyves, Please review last task for Let's encrypt SSL certificate obtaining at 35.187.97.192.nip.io
Thank you!

**Review by mrgreyves — APPROVED** [review]
Здравствуйте!) Апрув =)

**Pull request merged**